hugegraph-commons/pom.xml

```diff
@@ -124,6 +124,16 @@
         <module>hugegraph-rpc</module>
     </modules>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>1.18.30</version>
```

Comment:
Maybe we could add `<optional>true</optional>` and `provide` scope for it?

Reply:
the optional mean don't transfer dependency, for example, A -> B -> C, if B. use. optional true to c, the A will not reference C
so the optional in our code maybe don't importament